src/jinterface.jl

```diff
@@ -0,0 +1,69 @@
+"""
+Creates a type which incorporates the mesh attributes alongside the necessary values
+to solve finite element problems. It works by taking in a FEniCS mesh, working out
+the necessary values, and then creating an ordering of the internal/boundary nodes
+and applying the DirichletBC (via a function) to the respective nodes.
+"""
+type feMesh
+    n_nodes::Int64
+    n_elements::Int64
+    nodes::Array{Float64, 2}
+    internal_nodes::Array{Int64}
```

Comment:
Note that arrays need dimension to be concretely typed, so accessing this is type-unstable.

Reply:
Something like nodes::Array{Float64, 2::Int} or am I mistaken? Also for the Alias would it then be nodes::Matrix{Float64}? or preferably nodes::Matrix{AbstractFloat}?

Reply:
Just `Array{Float64,2}` or `Matrix{Float64}`. No assertions in there since that `2` is a value and not a type location.